src/styles.scss

```diff
@@ -150,6 +150,10 @@ mat-chip.mat-chip.mat-standard-chip {
 
 /* Generic custom styles */
 
+.mat-dialog-container {
```

Comment:
Not sure if this particular bit is to blame, but just in case: somewhere in this PR the event editors in mobile view became narrow enough in mobile view to produce an extra scrollbar, making it a bit awkward: 

https://user-images.githubusercontent.com/86378/106602245-05498400-655d-11eb-893c-a22e2f59530b.mp4

Reply:
@kvviecien thoughts?

Reply:
I'll try to figure out what's happening there and fix it. Expect a pr!

Reply:
So I didn't... I played with it for hours and I have no idea why the scroll appears there.
I'll try to rewrite some of it though. I think the HTML could be faulty, a lot's happening there.